NohanaImagePicker/AlbumListViewController.swift

```diff
@@ -37,14 +43,14 @@ class AlbumListViewController: UITableViewController, EmptyIndicatable, Activity
 
     weak var nohanaImagePickerController: NohanaImagePickerController?
     var photoKitAlbumList: PhotoKitAlbumList!
+    weak var delegate: AlbumListViewControllerDelegate?
 
     override func viewDidLoad() {
         super.viewDidLoad()
         if let nohanaImagePickerController = nohanaImagePickerController {
             view.backgroundColor = nohanaImagePickerController.config.color.background ?? .white
             title = nohanaImagePickerController.config.strings.albumListTitle ?? NSLocalizedString("albumlist.title", tableName: "NohanaImagePicker", bundle: nohanaImagePickerController.assetBundle, comment: "")
-            setUpToolbarItems()
```

Comment:
Could you remove unused function.

Reply:
This method is implemented in UIViewController extension, so I want to leave it as it is.